### ISSUE TITLE
RTC?

### ISSUE THREAD

**Issue by VanRillaerM** [Monkey0x9/Small-scale-remote-controlled-cargo-truck#2]
![image](https://user-images.githubusercontent.com/71498677/124314001-ff8ab580-db71-11eb-9029-1755a981bca2.png)

Is R6 een RTC? Zo ja, gelieve ook zo aan te duiden.

Figure 2. in datasheet
https://www.st.com/resource/en/datasheet/l6924u.pdf

**Comment by fredlord** [comment]
is geen RTC is een spannings deler voor refrence voltage geloof ik

**Comment by VanRillaerM** [comment]
nee, ga naar 8.1 in de datasheet waarvan ik de link heb gezet. tussen die spanningsdeler (waarvan een van de weerstanden een NTC of PTC is) wordt een verbinding gemaakt naar TH.
